apps/firecrawler/firecrawl_mcp/tests/conftest.py

```
@@ -37,6 +37,38 @@
+        self.debug_mode: bool = False
+        self.development_mode: bool = False
+        self.rate_limit_requests_per_minute: int = 60
+        self.rate_limit_requests_per_hour: int = 600
+
+    def is_valid(self) -> bool:
+        """Check if configuration is valid."""
+        return self.firecrawl_api_key is not None
+
+    def get_current_timestamp(self) -> str:
+        """Get current timestamp."""
+        from datetime import UTC, datetime
```

Pylint (3.3.8)

[convention] Import outside toplevel (datetime.UTC, datetime.datetime)

(C0415)